Add sentence about clearing events from Schema (#5248)
segmentio/segment-docs: src/connections/sources/schema/schema-unique-limits.md

![Clear your Schema data with Clear Schema History](images/schema_config_clear_schema.png)

> Clearing events from the Source Schema only clears it from the Segment interface. It does not impact the data sent to your destinations or storage warehouse. Once you clear the events, the Schema page will start to repopulate with new events.

Reviewer: ```suggestion
Clearing events from the Source Schema only clears them from the Segment interface. It does not impact the data sent to your destinations or warehouses. Once you clear the events, the Schema page starts to repopulate new events.
```
